WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsSelectPlugin.swift

```diff
@@ -37,8 +38,9 @@ struct InPersonPaymentsSelectPluginRow: View {
     }
 }
 
-struct InPersonPaymentsSelectPlugin: View {
+struct InPersonPaymentsSelectPluginView: View {
```

Comment:
Having View in the name makes the codebase much easier to navigate, no matter the example Apple set!

It's probably worth renaming the file to match as well.